scripts/install_prerequisites.sh

```diff
@@ -40,4 +40,5 @@ sudo apt-get install python3-yaml
 #    https://github.com/oppia/oppia/wiki/Troubleshooting
 sudo apt-get install python-matplotlib
 sudo apt-get install python3-matplotlib
+sudo apt-get install libbz2-dev
```

Comment:
Do we need to update the wiki page as well? (Considering, running `scripts/install_prerequisites.sh` is optional as per the wiki page, right? [not sure!])

Reply:
Link to wiki page: https://github.com/oppia/oppia/wiki/Installing-Oppia-(Linux;-Python-3)#install-prerequisites

Reply:
Yep, I had already added it. (see the command `sudo apt-get install ...`.)